src/core/libmaven/Fragment.cpp

```diff
@@ -201,6 +205,34 @@ void Fragment::buildConsensus(float productPpmTolr)
     this->consensus = consensusFrag; 
 }
 
+float Fragment::consensusRt()
+{
+    if (brothers.size() == 0) 
+		return this->rt;
```

Comment:
`this` is not required when using inside member functions. Code will start to look messy if you want to use a lot member variables.  Although, we do not have this in our guidelines, not sure if we should have it, but I generally avoid using `this` pointer, unless some sort of  `type casting` is required

Reply:
Yes, missed that. Lots of `this` usage in MAVEN. Will remove that.